evap/results/templates/distribution_widget.html

```diff
@@ -3,7 +3,9 @@
 
 {% spaceless %}
 {% if average is not None %}
-    <div class="d-flex">
+    <div class="d-flex" data-toggle="tooltip" data-placement="left" data-fallback-placement='["left", "bottom"]'
+        title="{% if weight_sum and evaluation.course.evaluation_count > 1 %}{% blocktrans with percentage=evaluation.weight|percentage:weight_sum %}This evaluation contributes {{ percentage }} to the final grade of the course.{% endblocktrans %}{% endif %}"
```

Comment:
This title shows also on individual questions.
Also (little nit): When the tooltip breaks to its bottom position, it looks weirdly positioned:
![Screenshot from 2020-03-09 20-18-59](https://user-images.githubusercontent.com/7093655/76249315-5bc21b80-6243-11ea-9255-6dd7e20d9487.png)

Reply:
i won't address the tooltip positioning (here)